packages/teleterm/src/ui/DocumentGateway/common.tsx

```diff
@@ -25,8 +25,21 @@ export const ConfigFieldInput: typeof FieldInput = styled(FieldInput)`
     }
 `;
 
+const ConfigFieldInputWithoutStepper: typeof ConfigFieldInput = styled(
+  ConfigFieldInput
+)`
+  ::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+  ::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+`;
```

Comment:
I have no idea why this works but nesting those styles under `input {}` didn't.

If I simply add `color: red` here, the style seems to be applied to the `input` itself rather than the whole component. @gzdunek maybe you have an idea what's going on here?

Reply:
Yes, it took me a while to notice that there was no closing bracket for the `input` selector in `ConfigFieldInput`. It was breaking the styles for the `ConfigFieldInputWithoutStepper`. Pushed a fix for that.